lib/recurly.js

```diff
@@ -289,6 +291,33 @@ export class Recurly extends Emitter {
       this.emit('ready');
     }
   }
+
+  /**
+   * Reports an event to the Reporter if it is available
+   *
+   * @private
+   */
+  report (...args) {
+    if (!this.reporter) return;
+    this.reporter.send(...args);
+  }
+
+  /**
+   * Binds important events to the EventDispatcher
+   *
+   * @private
+   */
+  bindReporting () {
+    if (!this.isParent) return;
+    ['focus', 'blur'].forEach(eventName => {
```

Comment:
Perhaps we can include these as a constant e.g. `EVENTS = ['focus', 'blur']`?

Reply:
I don't think it would be very useful to separate it since it's only used once and isn't really a concern of the class, but just of this one routine.